src/index.html

```diff
@@ -11,7 +11,125 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset name="personalInfo" class="group">
+        <legend>Personal information</legend>
+        <label class="input">
+          Surname:
+          <input
+            type="text"
+            name="surname"
+            required
+            autocomplete="off"
+          >
+        </label>
+        <label class="input">
+          Name:
+          <input
+            type="text"
+            name="name"
+            required
+            autocomplete="off"
+          >
+        </label>
+        <label class="input">
+          How old are you?
+          <input
+            type="number"
+            name="age"
+            min="1"
+            max="100"
+            value="12"
+          >
+        </label>
+        <label class="input">
+          Full date opf birth:
+          <input
+            type="date"
+            name="birth_date"
+            required
+          >
+        </label>
+        <label class="input">
+          I accept the term of the agreement
+          <input type="checkbox" name="agreement">
+        </label>
+      </fieldset>
+      <fieldset name="registration" class="group">
+        <legend>Registration</legend>
+        <label class="input">
+          E-mail:
+          <input
+            type="email"
+            name="email"
+            required
+            placeholder="email@example.com"
+          >
+        </label>
+        <label class="input">
+          Password:
+          <input
+            minlength="8"
+            maxlength="16"
+            type="password"
+            name="password"
+            required
+          >
+        </label>
+      </fieldset>
+      <fieldset name="interests" class="group">
+        <legend>An interesting fact about you!</legend>
+        <label class="input">Do you love cats?
+          <input
+            type="radio"
+            name="cats"
+            value="yes"
+          >
+          Yes
+          <input
+            type="radio"
+            name="cats"
+            value="no"
+          >
+          No
+        </label>
```

Comment:
Each radio button should have label, 'cause now it works like that


https://user-images.githubusercontent.com/84275496/214066954-daee965a-ce7d-413a-b786-10919e4710e4.mov